src/decisionimporter/agent/DecisionImporterAgent.php

```diff
@@ -128,7 +128,7 @@ function processUploadId($uploadId)
 
     $this->decisionImporterUtils->getOrCreateIds($this->reportData, $this);
 
-    $this->decisionImporterDataCreator->setUserId($this->setUserId);
+    $this->decisionImporterDataCreator->setUserId($this->userId);
```

Comment:
```suggestion
    $this->decisionImporterDataCreator->setUserId($userId);
```
Just for consistency as `$userId = $this->userId;` is already defined.

Reply:
Done the changes and committed.